modules/auxiliary/gather/browser_getprivateip.rb

```diff
@@ -0,0 +1,142 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'           => "Private IP Leakage to WebPage using WebRTC Function.",
+        'Description'    => %q(
+          This module exploits a vulnerability in browsers using well-known property of WebRTC (Web Real-Time Communications) which enables Web applications and sites to capture or exchange arbitrary data between browsers without requiring an intermediary.
+        ),
+        'License'        => MSF_LICENSE,
+        'Author'         => [
+          'Brendan Coles', #MSF Module
+          'Dhiraj Mishra'  #MSF Module
+          'Daniel Roesler'  #JS Code
+        ],
+        'References'     => [
+          [ 'CVE', '2018-6849' ],
+          ['URL', 'https://datarift.blogspot.in/p/private-ip-leakage-using-webrtc.html']
+        ],
+        'DisclosureDate' => 'Jan 26 2018',
```

Comment:
This should probably be the date the WebRTC IP leak technique was published. The first instance I'm aware of was from https://github.com/natevw some time around 2014.

Reply:
`Sep 5 2013`

https://github.com/natevw/ipcalf/commit/5c8f4382007f860562e4cf1a0d7bd0c53ddcac88